docs/source/libraries/lightgbm.rst

```diff
@@ -5,13 +5,13 @@ LightGBM
 
 LightGBM_ is a fast Gradient Boosting framework; it provides a Python
 interface. eli5 supports :func:`eli5.explain_weights`
-and :func:`eli5.explain_prediction` for ``lightgbm.LGBMClassifer``
-and ``lightgbm.LGBMRegressor`` estimators.
+and :func:`eli5.explain_prediction` for ``lightgbm.LGBMClassifer``, ``lightgbm.LGBMRegressor`` and ``lightgbm.Booster``  estimators. It is tested against LightGBM
+master branch.
```

Comment:
"It is tested against LightGBM master branch." is removed in https://github.com/TeamHG-Memex/eli5/commit/63e99182dc682bbf225355c80a24807396a747b6, sorry for the confusion. It shouldn't be re-added.

Reply:
got it, I will remove that line in next commit